Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml

```diff
@@ -0,0 +1,122 @@
+                            <div class="govuk-radios__item">
+                                <input type="radio" name="@nameof(Model.FeedbackOrContactOption)" id="@nameof(Model.FeedbackOrContactOption)" value="WholeSite" class="govuk-radios__input" @(Model.FeedbackOrContactOption == "WholeSite" ? "checked" : "")>
+                                <label for="location-0" class="govuk-label govuk-radios__label">The whole site</label>
+                            </div>
+
+                            <div class="govuk-radios__item">
+                                <input type="radio" name="@nameof(Model.FeedbackOrContactOption)" id="@nameof(Model.FeedbackOrContactOption)-1" value="SpecificPage" class="govuk-radios__input" @(Model.FeedbackOrContactOption == "SpecificPage" ? "checked" : "") aria-controls="conditional-SpecificPage" aria-expanded="true">
+                                <label for="location-1" class="govuk-label govuk-radios__label">A specific page</label>
+                            </div>
+
+                            <div class="govuk-radios__conditional govuk-body govuk-radios__conditional--hidden" id="conditional-SpecificPage">
+                                <div class="govuk-form-group">
+                                    <label for="link" class="govuk-label">Enter URL or name of page</label>
+                                    <input class="govuk-input" data-ga4-form-include-input="" id="@nameof(Model.UrlOfPage)" name="@nameof(Model.UrlOfPage)" spellcheck="false" type="text" value="@Model.UrlOfPage">
+                                </div>
+                            </div>                         
+
+                        </div>
+                    </fieldset>
+                </div>
+
+                <div class="govuk-form-group govuk-character-count" data-module="govuk-character-count" data-maxlength="10000">
+                    <h1 class="govuk-label-wrapper">
+                          @if (Model.Context == "FEEDBACK")
+                        {
+                            <label class="govuk-label govuk-label--l" for="@nameof(Model.Details)">
+                                What is your Feedback
+                            </label>
+                        }
+                        else if (Model.Context == "SUPPORT")
+                        {
+                            <label class="govuk-label govuk-label--l" for="@nameof(Model.Details)">
+                                What are the details?
+                            </label>
+                        }
+                    </h1>
+                    <div id="@nameof(Model.Details)-hint" class="govuk-hint">
+                        Do not include personal or financial information like your National Insurance number or credit card details
+                    </div>
+                    <textarea class="govuk-textarea govuk-js-character-count" id="@nameof(Model.Details)" name="@nameof(Model.Details)" rows="5">@Model.Details</textarea>
```

Comment:
The aria-describedby attribute seems to have been removed? It should be set as follows:
```suggestion
                    <textarea class="govuk-textarea govuk-js-character-count" id="@nameof(Model.Details)" name="@nameof(Model.Details)" rows="5" aria-describedby="@nameof(Model.Details)-hint @nameof(Model.Details)-info">@Model.Details</textarea>
```

Reply:
done